cocos/2d/CCLabel.cpp

```diff
@@ -932,6 +932,8 @@ void Label::createSpriteWithFontDefinition()
     texture->initWithString(_originalUTF8String.c_str(),_fontDefinition);
 
     _textSprite = Sprite::createWithTexture(texture);
+    //set camera mask using label's camera mask, because _textSprite may be null when setting camera mask to label
```

Comment:
` _textSprite may be null`: it is not correct, it is camera mask of _textSprite is null.